modules/modules.nix

```diff
@@ -203,7 +203,7 @@ let
     (loadModule ./services/xscreensaver.nix { })
     (loadModule ./services/xsuspender.nix { condition = hostPlatform.isLinux; })
     (loadModule ./systemd.nix { })
-    (loadModule ./targets/darwin.nix { condition = hostPlatform.isDarwin; })
+    (loadModule ./targets/darwin { condition = hostPlatform.isDarwin; })
```

Comment:
With this line, all `mkIf isDarwin` in submodules are no longer needed.

Reply:
Fixed.